packages/admin-ui/src/Accordion/components/AccordionTrigger.tsx

```diff
@@ -71,20 +100,7 @@ const AccordionTrigger = ({
                     </div>
                     <div className={"wby-flex wby-gap-xs"}>
                         {actions}
-
-                        {/* No need to show the separator if there are no actions and the item is not interactive. */}
-                        {actions && interactive && <AccordionItemAction.Separator />}
-
-                        {interactive && (
-                            <Icon
-                                size={"lg"}
-                                className={"wby-transition"}
-                                color={"neutral-strong"}
-                                data-role={"open-close-indicator"}
-                                label={"Open/close indicator"}
-                                icon={<KeyboardArrowDownIcon />}
-                            />
-                        )}
+                        {!isInteractable ? <OpenCloseIndicator /> : null}
```

Comment:
I saw we lost the separator. Shouldn't this still be here?

Another question: I can see before we were rendering OpenCloseIndicator if `interactive===true`, now it's the opposite?